tests/e2e/tx/bank.go

```diff
@@ -0,0 +1,86 @@
+package tx
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/cosmos/gaia/v23/tests/e2e/common"
+)
+
+func (h *Helper) ExecBankSend(
+	c *common.Chain,
+	valIdx int,
+	from,
+	to,
+	amt,
+	fees string,
+	expectErr bool,
+	opt ...common.FlagOption,
+) {
+	// TODO remove the hardcode opt after refactor, all methods should accept custom flags
```

Comment:
Not sure what this is referring to. Everything in this function seems fine to me?

Reply:
This is a legacy comment that someone else wrote, so I'm not sure exactly what it's referring to and all I did was move the func out

Reply:
I'm fine deleting the comment in the future then.